jetty12/src/test/java/ch/qos/logback/access/jetty/JettyBasicTest.java

```diff
@@ -68,7 +68,7 @@ public void getRequest() throws Exception {
 
     @Test
     public void eventGoesToAppenders() throws Exception {
-        URL url = new URL(JETTY_FIXTURE.getUrl());
+        URL url = new URL("http://localhost:" + RANDOM_SERVER_PORT + "/path/foo%20bar;param?query#fragment");
```

Comment:
optional: How about leaving it as it was to maintain the same entry point?

```suggestion
                URL url = new URL(JETTY_FIXTURE.getUrl() + "path/foo%20bar;param?query#fragment");
```

this will

Reply:
Thanks for your reply.

https://github.com/qos-ch/logback-access/pull/7/commits/53b341a4747c824c9bce5d35d93f1e06690d69c3 done.